RIS.js

```diff
@@ -57,83 +81,83 @@ var DEFAULT_EXPORT_TYPE = 'GEN';
 var DEFAULT_IMPORT_TYPE = 'journalArticle';
 
 var exportTypeMap = {
-	artwork:"ART",
-	audioRecording:"SOUND", //consider MUSIC
-	bill:"BILL",
-	blogPost:"BLOG",
-	book:"BOOK",
-	bookSection:"CHAP",
-	"case":"CASE",
-	computerProgram:"COMP",
-	conferencePaper:"CONF",
-	dictionaryEntry:"DICT",
-	encyclopediaArticle:"ENCYC",
-	email:"ICOMM",
-	film:"MPCT",
-	hearing:"HEAR",
-	journalArticle:"JOUR",
-	letter:"PCOMM",
-	magazineArticle:"MGZN",
-	manuscript:"MANSCPT",
-	map:"MAP",
-	newspaperArticle:"NEWS",
-	patent:"PAT",
-	presentation:"SLIDE",
-	report:"RPRT",
-	statute:"STAT",
-	thesis:"THES",
-	videoRecording:"VIDEO",
-	webpage:"ELEC"
+	artwork: "ART",
+	audioRecording: "SOUND", //consider MUSIC
+	bill: "BILL",
+	blogPost: "BLOG",
+	book: "BOOK",
+	bookSection: "CHAP",
+	case: "CASE",
+	computerProgram: "COMP",
+	conferencePaper: "CONF",
+	dictionaryEntry: "DICT",
+	encyclopediaArticle: "ENCYC",
+	email: "ICOMM",
+	dataset: "DATA",
```

Comment:
Note -- this doesn't check if dataset is available in the importing Zotero version: I didn't know how to do that here in the object and without that it'd be a mess to hack into the translator

Reply:
No, that's not a mistake. `ExportTypeMap` holds only bi-directional, unique mappings, and [is merged with](https://github.com/zotero/translators/blob/246071dd68bc3d7bf068cb8afff63e19f227408e/RIS.js#L165) `ImportTypeMap` (see also the comments above each of them).
So this does impact import -- could we do the item type switch all the way at the end in post-processing? That seems cleanest, I think.

Reply:
Cool, actually put the code all the way at the top so the item type would be treated correctly during import throughout. Works nicely best I could test.